src/Raven.Server/Documents/Handlers/SubscriptionsHandler.cs

```diff
@@ -177,6 +178,8 @@ public async Task Delete()
             var subscriptionName = GetQueryStringValueAndAssertIfSingleAndNotEmpty("taskName");
 
             await Database.SubscriptionStorage.DeleteSubscription(subscriptionName, GetRaftRequestIdFromQuery());
+            
+            Database.SubscriptionStorage.RaiseNotificationForTaskRemoved(subscriptionName);
```

Comment:
I don't like using SubscriptionStorage and TaskRemoved in single line. 
RaiseNotificationForTaskRemoved suggests that any ongoing task was removed, but this is in SubscriptionStorage
 so maybe it is only for subscriptions, and so on? 

So for me:
Move this event one level up and fire that with every ongoing tasks (when adding / deleting)
or rename event name with: OnSubscriptionTaskRemoved

Reply:
In order to follow our convention let's keep that events as is in `SubscriptionStorage`. We put such events in dedicated classe - `ETLLoader`, `ReplicationLoader` etc For data subscriptions `SubscriptionStorage` seems to be best candidate.

We can rename the event to make it even more clear (although for me it's quite obvious that this is about subscription task because it it in `SubscriptionStorage`)

Reply:
In [this comment](https://github.com/ravendb/ravendb/pull/11766#discussion_r576031205) the request was to drop the word 'Subscription' to avoid repeating.

Reply:
Good point - it was requested by Marcin :) Let's leave it as is IMO